src/app/ugur-my-component/ugur-my-component.css

```diff
@@ -0,0 +1,10 @@
+mat-card {
+    max-width: 400px;
+    margin: 20px auto;
+    padding: 16px;
+  }
+  
+  mat-list-item {
+    font-size: 16px;
```

Comment:
Or changing styles onsie twosie... I'd rather all components use the default MD styles